src/main/java/com/github/technus/tectech/thing/casing/SpacetimeCompressionFieldCasing.java

```diff
@@ -0,0 +1,118 @@
+package com.github.technus.tectech.thing.casing;
+
+import static com.github.technus.tectech.thing.casing.GT_Block_CasingsTT.texturePage;
+import static com.github.technus.tectech.util.CommonValues.EOH_TIER_FANCY_NAMES;
+import static net.minecraft.util.EnumChatFormatting.*;
+
+import com.github.technus.tectech.thing.CustomItemList;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import gregtech.api.enums.Textures;
+import gregtech.api.objects.GT_CopiedBlockTexture;
+import gregtech.api.util.GT_LanguageManager;
+import gregtech.common.blocks.GT_Block_Casings_Abstract;
+import gregtech.common.blocks.GT_Material_Casings;
+import java.util.List;
+import net.minecraft.client.renderer.texture.IIconRegister;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.IIcon;
+import net.minecraft.world.IBlockAccess;
+
+@SuppressWarnings("SpellCheckingInspection")
+public class SpacetimeCompressionFieldCasing extends GT_Block_Casings_Abstract {
+    private static IIcon textureTier0;
+    private static IIcon textureTier1;
+    private static IIcon textureTier2;
+    private static IIcon textureTier3;
+    private static IIcon textureTier4;
+    private static IIcon textureTier5;
+    private static IIcon textureTier6;
+    private static IIcon textureTier7;
+    private static IIcon textureTier8;
+    private static final int maxBlockTier = 9;
+
+    private static final byte START_INDEX = 16;
```

Comment:
Do all these casings work? It seems weird that they all are written into the same spot in the texture page

Reply:
Yep

Reply:
Because someone wrote `i+base` instead of `base+i`.